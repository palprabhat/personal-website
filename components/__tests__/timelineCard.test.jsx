import { render } from "@testing-library/react";
import TimelineCard from "../timelineCard";

describe("TimelineCard Component", () => {
  it("matches snapshot", () => {
    const { baseElement } = render(<TimelineCard />);

    expect(baseElement).toMatchSnapshot();
  });
});
