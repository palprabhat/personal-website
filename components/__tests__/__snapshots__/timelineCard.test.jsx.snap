// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineCard Component matches snapshot 1`] = `
<div>
  <div
    class="timeline-card ml-2 md:ml-0 flex md:flex-row-reverse"
  >
    <div
      class="relative work w-4/5 md:w-2/5 mx-auto md:mr-0 bg-primary-200 rounded-xl p-4 text-left mb-4 "
    >
      <div
        class="text-right text-gray-700 text-sm"
      />
      <h3 />
      <h4 />
      <ul
        class="pt-4 list-disc pl-4"
      />
    </div>
  </div>
</div>
`;
